PhysicsTools/NanoAOD/python/photons_cff.py

```diff
@@ -338,7 +352,6 @@ def make_bitmapVID_docstring(id_modules_working_points_pset):
                               cutBased_Fall17V1Bitmap = None,
```

Comment:
also the cutBased_Fall17V1 can be removed, no ?
Fall17V2 should be the recommended for the Run2 legacy

Reply:
yes, I missed it while rebasing, pushing now